packages/components/src/highlight-cards/mobile-highlight-cards.tsx

```diff
@@ -8,16 +8,31 @@ type MobileHighlightCardProps = {
 	count: number | null;
 	previousCount?: number | null;
 	icon?: React.ReactElement;
+	preformattedValue?: string;
 };
 
-function MobileHighlightCard( { heading, count, previousCount, icon }: MobileHighlightCardProps ) {
-	// We require at minimum a heading and a count.
-	const isValidHighlight = count !== null && heading.length > 0;
-	if ( ! isValidHighlight ) {
+function MobileHighlightCard( {
+	heading,
+	count,
+	previousCount,
+	icon,
+	preformattedValue,
+}: MobileHighlightCardProps ) {
+	// We'll accept a count or a preformatted string as our value.
+	// If both are provided, we'll use the preformatted string for display.
+	const hasValidDisplayValue =
+		( count !== null && count !== undefined ) ||
```

Comment:
theoretically `count` wouldn't be be `undefined`; otherwise it wouldn't pass TS checks. It seems like you are trying to test whether the value is a valid number - I see pps using `Number.isFinite()` for the purpose.